### PR TITLE
Fixed: #1 - Exposes headers as public

### PR DESCRIPTION
**Pull request by awgeorge**
Fixed: #1 - Exposes headers as public
Fixed: Error in documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,8 +36,8 @@ BOOL crashedOnPriorLaunch = ...
 id<UBApplicationStartupReasonReporterPriorRunInfoProtocol> runInfo = ...
 
 // Initialize the startup reason reporter
-UBApplicationStartupReasonReporter *applicationStartupReasonReporter = [[UBApplicationStartupReasonReporter alloc] initWithNotificationCenter:[NSNotificationCenter defaultCenter]
-        previousRunDidCrash:self.crashReporter.crashDetected
```
Comment (artman) [inline README.md:40]: This makes it less obvious that you'd need to pull this information from your crash reporting tool.

File: StartupReasonReporter/Info.plist [hunk 2]
```diff
@@ -17,7 +17,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
```
Comment (artman) [inline StartupReasonReporter/Info.plist:20]: Please revert this change.

**Review by artman — COMMENTED** [review]
Thank you for your contribution. You would need to diff this up with a email address that matches a Github user in order to be able to sign the CLA.

File: README.md (hunk 1 above)
Comment (awgeorge) [reply README.md:40]: This change matches the swift. It also matches the fact that you've called the local variable `crashedOnPriorLaunch` three lines above.

File: StartupReasonReporter/Info.plist (hunk 2 above)
Comment (awgeorge) [reply StartupReasonReporter/Info.plist:20]: Sure, you don't want the version number to match with the released carthage version?

**Comment by awgeorge** [comment]
I've now added the email to my account.

File: README.md (hunk 1 above)
Comment (artman) [reply README.md:40]: Sorry, did a too hurried review. Right you are.

File: StartupReasonReporter/Info.plist (hunk 2 above)
Comment (artman) [reply StartupReasonReporter/Info.plist:20]: Yeah, the version numbers mismatch, but we'll fix it when we release. Since it's already broken this is fine.
